lib/theme/app_colors.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  static const blackRussian = Color(0xFF15151A);
```

Comment:
@luongvo Interesting, but is that really necessary for `colors`? 

From my understanding, `flutter_gen` is used for **safety** to ensure the correct path is given.

But since we don't use path for the colors, my approach looks more convenient 💭 

What do you think? 🙏

Reply:
@Tuubz We prefer using `colors.xml` because it's more familiar with Android dev, and it's more easy when migrating a project to Flutter as we can reuse the xml file. 

Anw, I just want to suggest a new way as configurable. Your way works too, no biggles.